### PR TITLE
Feat: Show Idea Card Info

### LINKED ISSUE #962: Idea cards: show how many comments & interactions
Right now it is not possible to see how popular an idea is. That's why the number of comments and users that joined the group chat should be displayed on the idea card. 
If the number of comments and/or likes is 0 that one should not be displayed. 

Image

### PR DESCRIPTION
**Pull request by JoshKornfeld**
## Description

Closes #962
This PR displays the number of chat participants in the group chat of the idea and the number of comments left on it.
See below for images of the changes (I don't have the category icons in my backend).

Backend changes:
- Updated serializer to include number of participants in the group chat of the idea, number is obtained via utility functions.

Frontend changes:
- IdeaPreviews has a number reflecting the number of comments/participants, alongside an icon. The number and icon will not be displayed if it is  < 1. 
- IdeaPreviews with an image will have the information displayed ontop of the image.
- IdeaPreviews without an image will have the information displayed underneath the summary text.

Before:

77b4f7b3dcdb6677dd7fe454f2873f2d

After: 

Screenshot from 2022-11-11 14-14-29

## Test plan

1. Go to a Climate Hub
2. Click the ideas tab
3. See the previews now have the additional feature

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)

### CODE REVIEW

File: backend/ideas/serializers/idea.py [hunk 1]
```diff
@@ -40,6 +45,8 @@ class IdeaMinimalSerializer(serializers.ModelSerializer):
     organization = serializers.SerializerMethodField()
     url_slug = serializers.SerializerMethodField()
     hub_shared_in = serializers.SerializerMethodField()
+    number_of_comments = serializers.SerializerMethodField()
```
Comment (JoshKornfeld) [inline backend/ideas/serializers/idea.py:48]: Perhaps it is sensible to make a new serializer that inherits this one and add the 2 new fields to that one?

File: backend/ideas/utility/idea.py [hunk 2]
```diff
@@ -149,3 +151,20 @@ def get_idea_short_description(idea: Idea, language_code: str) -> str:
...
+        is_abusive=False,
+        deleted_at__isnull=True,
+        parent_comment=None,
+    ).count()
+
+
+def get_number_of_idea_participants(idea: Idea):
+    chats = MessageParticipants.objects.filter(name=idea.name)
+    for chat in chats:
+        user_count = Participant.objects.filter(chat=chat, is_active=True).count()
```
Comment (ddhanesha) [inline backend/ideas/utility/idea.py:168]: Can you update this query to something like this,

```python
chat_user_count = Participant.objects.filter(chat__name=idea.name, is_active=True).count()
```

This makes one less query to our database and avoids a for loop you have in your code.

**Review by ddhanesha — CHANGES_REQUESTED** [review]

File: backend/ideas/utility/idea.py (hunk 2 above)
Comment (JoshKornfeld) [reply backend/ideas/utility/idea.py]: Thanks for the suggestion, changed it :)

**Comment by JoshKornfeld** [comment]
@ddhanesha ready for another review :+1:

**Review by ddhanesha — APPROVED** [review]
Idea card info is working properly. Approved!

**Pull request merged**